Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -28,16 +28,31 @@ module.exports.Quantity = class Quantity extends Expression
     if other instanceof Quantity and other.unit == @unit then @value > parseFloat other.value else null
 
   before: (other) ->
-    if other instanceof Quantity and other.unit == @unit then @value < parseFloat other.value else null
+    if other instanceof Quantity and other.unit == @unit 
+      @value < parseFloat other.value 
+    else if other instanceof Quantity and other.unit of time_units and @unit of time_units
+      thisSmallestDuration = smallestDuration(@)
+      otherSmallestDuration = smallestDuration(other)
+      thisSmallestDuration < otherSmallestDuration
+    else null
```

Comment:
Agreed regarding the other comparison operators.  While we _should_ support other operations as well (addition, etc) -- I think I'd merge in this PR without them, as it does represent an incremental improvement.  But for consistency, I think we definitely need to support _all_ comparison operators -- it's confusing to support only `before`.